src/components/FilterSelectorEntry.tsx

```diff
@@ -0,0 +1,35 @@
+import React, {MouseEvent} from 'react';
+import "../stylesheets/filterbuttons.scss"
+import {FontAwesomeIcon} from "@fortawesome/react-fontawesome";
+import {FilterButtonProps} from "./FilterButton";
+import {IconProp} from "@fortawesome/fontawesome-svg-core";
+
+
+export interface FilterSelectorEntryProps {
+    text: string,
+    icon: IconProp,
+    onClick: (text: string) => void
+}
+
+
+export class FilterSelectorEntry extends React.Component<FilterSelectorEntryProps> {
```

Comment:
Do we need a whole component for these? I think we could (in FilterSelector inside the `ul` ) do something like this:
```
selectorLists.map(listItem => {
    return (
        <button className = "contextFilterSelectorEntry" onClick={this.handleClick.bind(this)}>
            <span className="button-icon"><FontAwesomeIcon icon={this.props.icon} fixedWidth/></span>
            {this.props.text}
        </button>)})
```
Just saying because it seems like FilterSelectorEntry only needs to handle onclick, which the `<button>` element already handles.